Assets/Scenes/Columbia/main.gd

```diff
@@ -9,6 +9,7 @@ enum client_packets {
 	ROD_SELECT_UPDATE = 11,
 	USER_LOGIN = 12,
 	SYNCHRONIZE = 14,
+	CHAT = 15,
```

Comment:
it won't actually cause any problems, but we'd like to avoid having a client and a server packet with the same ID, just so that if you say "packet x" you don't also need to mention if it's a client packet or server packet